src/components/App/App.spec.js

```diff
@@ -1,21 +1,20 @@
 import React from 'react';
 import { shallow } from 'enzyme';
 import App from './App';
-import Calculator from '../Calculator/Calculator';
+
+jest.mock('../Calculator/Calculator', () => 'Calculator')
 
 describe('App', () => {
-    let wrapper;
+    let subject;
     beforeEach(() => {
-        wrapper = shallow(<App />);
+        subject = shallow(<App />);
     })
 
-    it ('should render correctly', () => expect(wrapper).toMatchSnapshot())
-    
-    it ('should render a div', () => {
-        expect(wrapper.find('div').length).toEqual(1);
+    it ('renders a div with className "app-container"', () => {
+        expect(subject.find('div').length).toEqual(1);
     });
 
-    it ('should contain a Calculator Element', () => {
-        expect(wrapper.containsMatchingElement(<Calculator />)).toEqual(true)
+    it ('contains a Calculator Element', () => {
```

Comment:
```suggestion
    it ('contains a Calculator component', () => {
```
Pedantic, but more correct 😅